lib/internal/per_context/primordials.js

```diff
@@ -419,7 +419,13 @@ primordials.SafeWeakMap = makeSafe(
 primordials.SafeSet = makeSafe(
   Set,
   class SafeSet extends Set {
-    constructor(i) { super(i); } // eslint-disable-line no-useless-constructor
+    constructor(i) {
+      super(
+        i && primordials.isArray(i) ?
```

Comment:
There’s no such thing as `primordials.isArray`, I think thou meant `ArrayIsArray`.

```suggestion
        ArrayIsArray(i) ?
```
Also add `ArrayIsArray` to the `const { ... } = primordials` block above: <https://github.com/nodejs/node/blob/5077ea41491d555b44675f86502d4a91850dcbfd/lib/internal/per_context/primordials.js#L269-L310>